course_info/templates/course_info/partials/remove-course-membership-confirmation.html

```diff
@@ -0,0 +1,28 @@
+{% verbatim %}
+<div class="modal-header">
+	<button type="button" class="close" ng-click="$dismiss('cancel')"
+          id="modalClose">
+    &times;
+  </button>
+	<h4 class="modal-title">Confirm Remove User</h4>
+</div>
+<div class="modal-body">
+  <p>Are you sure you want to remove</p>
+  <div style="border-style: solid; border-width: 1px 0px 1px 0px; border-color: #ccc; padding:.5em; margin-top:1em; margin-bottom:1em;">
```

Comment:
nope.  that same style is used in a few other places...i think there's one spot where it varies, maybe in the modal partial?